### ISSUE TITLE
поиск в find

### ISSUE THREAD

**Issue by amelice** [bem/bem-forum-content-ru#742]
Метод find в колбеке состояния(capture) ищет элемент от селектора, который указан в setCaptureElements?

**Comment by tadatuta** [comment]
cc @arikon @SevInf @sipayRT

**Comment by sipayRT** [comment]
нет, он не привязан к селектору из `setCaptureElements`. Пример ситуации - нужно сделать скрин попапа, который откроется только после клика на кнопку. В этом случае кнопка будет вне попапа

**Comment by amelice** [comment]
Спасибо, понятно. :)
